AmethystAPI/src/amethyst/runtime/patchManager/PatchManager.cpp

```diff
@@ -0,0 +1,55 @@
+#include "PatchManager.h"
+#include "../../Log.h"
```

Comment:
Could this be switched to absolute paths

Reply:
Yeah. Although what is the reason if you dont mind me asking

Reply:
my brain doesn't work in relative only absolute... idk

Reply:
Fair enough just didnt know if there was some strange C++ isms i didnt know about